### DISCUSSION TITLE
Remove default node from room listing

### DISCUSSION THREAD

**Discussion by dumbmoron** [netdata/netdata#16817] [Q&A] [answered]
I have a setup where I have a central node, to which I stream from various different servers. I don't want to run monitoring on the central node, and thus I turned off all the plugins and have no charts to display for it. Is there any way to hide this node from the rooms listing, since it has no monitoring set up?

**Comment by ilyam8** [comment, marked as answer]
Hey, @dumbmoron. No, that is not possible.
